substrate/client/rpc-spec-v2/src/transaction/transaction_broadcast.rs

```diff
@@ -106,17 +108,19 @@ where
 		// The unique ID of this operation.
 		let id = self.generate_unique_id();
 
+		// There is nothing we could do with an extrinsic of invalid format.
```

Comment:
Maybe improve this comment to something like that spec states because this logic was not super obvious to me:

> The JSON-RPC server might check whether the transaction is valid before broadcasting it. If it does so and if the transaction is invalid, the server should silently do nothing and the JSON-RPC client is not informed of the problem. Invalid transactions should still count towards the limit to the number of simultaneously broadcasted transactions.